packages/conventional-gitmoji/src/config/rewrites.ts

```diff
@@ -8,63 +8,69 @@ type RewritesProps = {
   releaseNotes?: boolean | null
   semver?: 'breaking' | 'feature' | 'fix' | 'major' | 'minor' | 'patch' | null
   to: string
+  title?: string
 }
 
 /* eslint-disable sort-keys */
 const rewrites: RewritesProps[] = [
   { from: 'art', to: 'style' },
-  { from: 'zap', to: 'perf' },
+  { from: 'zap', to: 'perf', title: 'Performance' },
   { from: 'fire', to: 'prune' },
   { from: 'bug', to: 'fix', branch: 'fix' },
   { from: 'ambulance', to: 'hotfix', branch: 'hotfix' },
-  { from: 'sparkles', to: 'feat', branch: 'feature' },
+  { from: 'sparkles', to: 'feat', branch: 'feature', title: 'Feature' },
   { from: 'memo', to: 'docs' },
   { from: 'rocket', to: 'deploy' },
   { from: 'lipstick', to: 'ui' },
   { from: 'tada', to: 'init', semver: 'patch' },
   { from: 'white-check-mark', to: 'test', branch: 'test' },
   { from: 'lock', to: 'security' },
-  { from: 'bookmark', to: 'release', branch: 'release', releaseNotes: false },
+  { from: 'bookmark', to: 'release', branch: 'release', releaseNotes: true },
```

Comment:
We can probably remove `releaseNotes` completely and instead have it only come from a configuration setting within `release.config` going forward.

Keeping for now for historic purposes until there is time to add more pizzazz to the configuration.